setup.cfg

```diff
@@ -37,7 +39,7 @@ edit_on_github = True
 github_project = astropy/astrowidgets
 # install_requires should be formatted as a comma-separated list, e.g.:
 # install_requires = astropy, scipy, matplotlib
-install_requires = astropy, ginga, pillow, ipywidgets, ipyevents>=0.4.0
+install_requires = astropy, ginga, pillow, ipywidgets>=7.5, ipyevents>=0.6.3, jupyterlab>=1, aggdraw
```

Comment:
I feel like requiring jupyterlab on install is overkill, given that the widget can run without it?

Reply:
On the other hand, end-of-life of classic notebook isn't too far off (hard date has not been announced as far as I know), and it is an easy install (wheels available). Can remove it if you prefer though, but will need to update the install instructions then.

Reply:
Hmm... I guess it doesn't hurt to leave it in for now.